server/api/index.js

```diff
@@ -15,6 +15,7 @@ const errorHandler = (err, req, res, next) => {
   if (res.headersSent) {
     return next(err);
   }
+  console.error(err); // eslint-disable-line no-console
```

Comment:
Since I see increasingly more console.log/error checkin in to the repo. I think it may be official allow usage of console?

In the same time, we add strip console.log in webpack pipeline to avoid shiping console.log to client side script.

Does it make sense to you? @nwingt

Reply:
Stripping console.log/error in production make sense for me
I think a logging library is needed, do you have any suggestion @rickmak